test/libsolidity/semanticTests/fallback/fallback_return_data.sol

```diff
@@ -0,0 +1,14 @@
+contract A {
+    fallback (bytes calldata _input) external returns (bytes memory) {
+        return _input;
+    }
+    function f() public returns (bool, bytes memory) {
+        (bool success, bytes memory retval) = address(this).call("abc");
+        return (success, retval);
+    }
+}
+// ====
+// compileViaYul: also
+// EVMVersion: >=byzantium
+// ----
+// f() -> 0x01, 0x40, 0x03, 0x6162630000000000000000000000000000000000000000000000000000000000
```

Comment:
This is just how the auto-correct function of isotest formats it. If you want, you can check why it is done like that and fix it.

Reply:
For me isoltest actually formats it like this:
```
// f() -> 1, 0x40, 3, 44048180597813453602326562734351324025098966208897425494240603688123167145984
```

But if you write "abc", it won't reformat it as long as the test passes. I'm suggesting it mostly because I've seen you do it in another .sol test in this PR and because it's more readable that way.

Anyway, this is not very important. I'm fine with leaving it as is too. "abc" is pretty easy to recognize in ASCII. Would be harder with a longer string.

Reply:
Weird...